src/event-schemas/cumulative-layout-shift-event.json

```diff
@@ -11,7 +11,36 @@
         },
         "value": {
             "type": "number",
-            "description": "Value of the cls metric"
+            "description": "Largest burst of unexpected layout shifts during a page's lifespan"
+        },
+        "attribution": {
```

Comment:
Note: Individual attributes are not required by design. They are sourced from the web vitals package, which does not guarantee that every attribute will be included